File: homeworks/hw3/Models/Square.java
package homeworks.hw3.Models;

public class Square extends Figure {

    private double side;

    public Square(double side) {
        this.side = side;
    }

    public double getPerimeter() {
        return 4 * side;
    }

    public double getArea() {
        return side * side;
    }

    public double getSide() {
        return side;
    }

}

File: finaltask/models/square.py
from finaltask.models.figure import Figure


class Square(Figure):

    def __init__(self, side):
        self.side = side

    def get_perimeter(self):
        return 4 * self.side

    def get_area(self):
        return self.side * self.side

    def get_side(self):
        return self.side
